src/ui/Windows/res/PasswordSafe3.rc2

```diff
@@ -262,8 +262,18 @@ BEGIN
   IDS_DEFAULTPWLENGTHTOOSMALL "Password length is less than sum of 'at least' constraints"
   IDS_CANTSPECIFYMINNUMBER "Sorry, can't specify a minimum number of characters with 'pronounceable' option. Specified values ignored."
   IDS_OWNSYMBOLSMISSING    "If you specify using your own symbol list, you cannot leave it empty. Unselecting use of symbols."
-  IDS_EXCLUDE_FROM_SCR_CAP_OVERRIDE_WARNING_BEFORE "Screen capture protection disabled by command line.\n\nContact support if this is unexpected.\n\nYou can click OK to continue in %d seconds."
-  IDS_EXCLUDE_FROM_SCR_CAP_OVERRIDE_WARNING_AFTER  "Screen capture protection disabled by command line.\n\nContact support if this is unexpected.\n\nClick OK to continue."
+
+  IDS_SCRCAP_EXCLUDE_OVERRIDE_WARNING_BEFORE_INVALID "Screen capture protection disabled by unknown cause (state error).\n\nContact support if this is unexpected.\n\nYou can click OK to continue in %d seconds."
```

Comment:
Is this what's shown if the state isn't console/citrix/rdp?
If so, at least we need to show the value, otherwise how's support supposed to handle this?

Reply:
Are you citing the "*_INVALID" string? That is a placeholder that should generally never be shown. It is shown when there is an internal program failure due to bad enumeration value (see source which uses this resource).

Reply:
The `UINT nId = nIdFirst - 1;` in below is effectively defaulting to the *_INVALID messages.
```
UINT ThisMfcApp::ResolveAllowScreenCaptureStateResourceId(UINT nIdFirst) const
{
  // nIdFirst is the first known valid state.
  // nIdFirst - 1 should be a resource indicating a program/state error.
  // From nIdFirst through to the last state (nIdFirst + MaxState - 1),
  // the resource should reflect within itself the indicated state.
  UINT nId = nIdFirst - 1;
  if (m_allowScreenCaptureState >= Disallowed || m_allowScreenCaptureState < MaxState)
    nId = nIdFirst + m_allowScreenCaptureState;
  return nId;
}
```

Reply:
Sorry I wasn't clear. My comment was about the text itself. Instead of "...unknown cause (state error)" I think something like "...unknown cause (state=%d)" would give "support" a bit more info about the problem.
Otherwise, what would support be expected to do???

Reply:
> Sorry I wasn't clear. My comment was about the text itself. Instead of "...unknown cause (state error)" I think something like "...unknown cause (state=%d)" would give "support" a bit more info about the problem. Otherwise, what would support be expected to do???

The value won't be helpful generally speaking because it will be caused by memory corruption, failing memory, buffer overrun, or a program bug.

That resource message is a placeholder for out of range enum, an _extraordinarily rare_ situation _not outweighing_ the value of consistency in resource string message handling, if that makes sense. 

How code handles invalid enums can vary... it's not uncommon for some default behavior to be taken upon such detection... but in this case, I prefer to display a message that is different without overengineering it.

I mean, we could add code to detect if the resource message contains %d and, if so, include the enum value in there... but I feel that increases complexity in this area for little if any value.

Reply:
OK, I'm convinced - let's leave this as-is.